src/pages/locations.tsx

```diff
@@ -87,21 +87,23 @@ const LocationsPage = () => {
                 )}
               </Grid>
               <Grid item xs={12} sm={12} md={3} lg={3}>
-                <h2>Next hour</h2>
+                <h2>Next hour forecast</h2>
                 { isHourlyForecastLoading ? (
                   <Loading />
                 ) : (
                   <CurrentHourForecast forecast={forecastDataHourly} idx={forecastStartingIndex} />
                 )}
               </Grid>
-              <Grid item xs={12} sm={12} md={3} lg={3}>
-                <h2>Tide</h2>
-                {isTideDataLoading ? (
-                  <Loading />
-                ) : tideData && (
-                  <DailyTide {...tideData} />
-                )}
-              </Grid>
+              {(location as { station_id?: string })?.station_id && (
```

Comment:
I realized that not all locations have a tide station; eg; buoys in the great lakes region. Those particular pages threw an error as the object key `station_id` was null.